__tests__/api.test.js

```diff
@@ -139,34 +139,66 @@ describe("GET/api/articles/:article_id/comments", () => {
   });
   it("status 200, responds with an empty array if the article doesn't have any comments", () => {
     return request(app)
-      .get("/api/articles/2/comment")
+      .get("/api/articles/2/comments")
       .expect(200)
       .then(({ body }) => {
         expect(body.comments).toEqual([]);
       });
   });
   it("status 200, comments Sorted by Date", () => {
     return request(app)
-      .get("/api/articles/1/comment")
+      .get("/api/articles/1/comments")
       .expect(200)
       .then(({ body }) => {
         expect(body.comments).toBeSortedBy("created_at", { descending: true });
       });
   });
   it("status 400, responds with a message when the id datatype is not valid", () => {
     return request(app)
-      .get("/api/articles/banana/comment")
+      .get("/api/articles/banana/comments")
       .expect(400)
       .then(({ body }) => {
         expect(body.msg).toBe("Bad Request");
       });
   });
   it("status 404, responds with a message if id doesn't exist", () => {
     return request(app)
-      .get("/api/articles/8500/comment")
+      .get("/api/articles/8500/comments")
       .expect(404)
       .then(({ body }) => {
         expect(body.msg).toBe("Not Found");
       });
   });
 });
+describe("POST/api/articles/:article_id/comments", () => {
```

Comment:
We could definitely be more thorough with the testing here. Some examples to test for would be:
- What if the id in the url is, for example,  "ten" instead of 10
- what if the id is valid but doesn't exist eg 99999
- what if the username doesn't exist